charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ template "zookeeper-operator.fullname" . }}
+  namespace: {{ $.Release.Namespace }}
```

Comment:
Is the `$` not redundant here, given that we're already scoped at the root (`.`)?

Reply:
done, thanks